portfolio/src/main/webapp/script.js

```diff
@@ -72,14 +72,19 @@ function loadMap() {
         position: desiredLocationLatLng, 
         map: map,
         animation: google.maps.Animation.DROP,
-        title: 'Travel Destinations'
+        title: 'Travel Destinations: Click me to see more!'
       });
+    const infoWindow = new google.maps.InfoWindow({
+      content: contentString
```

Comment:
OOC, I don't know how this infoWindow works and i also don't know where is this `contentString` comping from.

Reply:
oops! looks like I deleted the contentString variable in a previous commit :/